internal/exporter/prometheus/collector/power_collector_test.go

```diff
@@ -309,6 +326,86 @@ func TestPowerCollector(t *testing.T) {
 		})
 	})
 
+	t.Run("Process Metrics Node Name Label", func(t *testing.T) {
+		metrics, err := registry.Gather()
+		assert.NoError(t, err)
+
+		for _, metric := range metrics {
+			if metric.GetName() == "kepler_process_cpu_joules_total" {
+				for _, m := range metric.GetMetric() {
+					nodeName := valueOfLabel(m, "node_name")
+					assert.Equal(t, "test-node", nodeName, "Process metrics should have node_name label")
+				}
+			}
+			if metric.GetName() == "kepler_process_cpu_watts" {
+				for _, m := range metric.GetMetric() {
+					nodeName := valueOfLabel(m, "node_name")
+					assert.Equal(t, "test-node", nodeName, "Process metrics should have node_name label")
+				}
+			}
```

Comment:
suggestion: refactor this into a helper function that assert that `assertMetricLabelValue(t, name, label, value)` we have something similar called `asserrtMainMetricValue` which I think can be replaced as well.